package.json

```
@@ -3,8 +3,21 @@
     "version": "3.1.2",
     "description": "Decode JWT tokens, mostly useful for browser applications.",
     "main": "build/jwt-decode.cjs.js",
-    "module": "build/jwt-decode.esm.js",
-    "types": "index.d.ts",
+    "module": "build/jwt-decode.mjs",
+    "types": "index.d.mts",
```

This field should match the `main` field.

```suggestion
    "types": "index.d.ts",
```